HW02_B_Mimic_with code.Rmd

```diff
@@ -0,0 +1,122 @@
+---
+title: "HW02_B_Graph-Mimic"
+author: "Sherzod Tokamov"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+
+```{r load libraries, echo = TRUE, include=TRUE}
+library(ggplot2)
+library(magrittr) #so I can do some piping
+data(diamonds)
+data(mpg)
+data(iris)
+theme_set(theme_bw()) #I'll give you this one, you can set the theme individually for graphs
+#or you can set the theme for all the graphs using theme_set()
+#theme_bw() is best theme (IMO)
+
+#for graph 3:
+library(ggrepel)
+
+
+```
+
+## HW02 Part B
+
+For this part of the HW, the goal is to try to recreate the graphs I make from scratch. I will only provide the MD, not the actual code I used to create it besides which data I use to create it. The rest will be up to you. 
+
+Try for all 4, but if you are struggling don't worry about it. Try your best for each, if you don't get everything that's what the peer-review is for. :smile:
+
+
+### Graph 1
+```{r, echo = T, include = TRUE}
+data("diamonds")
+#hint think about the *position* the bars are in...
+```
+
+
+Using the diamonds dataset, make this graph:
+```{r graph1 code, echo=FALSE}
+#Create a bar graph using geom_bar and specify position="dodge" to have bars side by side.
+ggplot(diamonds, aes(x=cut, fill = clarity)) +
+    geom_bar(stat="count", position = "dodge") +
```

Comment:
All my code was the same for this except the stat="count" in geom_bar. I'm not sure what this function does and when I applied it to my code to see the graph looked the same. Good job, looks exactly the same as the template